insert pross and dssp data into database

NicolasBourassin opened the pull request:
flask-sqlAlchemy to insert new data into database

NicolasBourassin committed:
insert pross and dssp data into database

CharlottePerin reviewed (commented):
appeler la fonction dssp/pross directement depuis le insert Annotation (donc virer les appels de la fonction upload non?)
Pourquoi insérer des infos dans la base de données revient UNIQUEMENT à insérer des annotations? Où sont les angles etc...

NicolasBourassin committed:
Correct some (stupid) errors about data insertion

- function insert() put into upload()
- explicit call of model.Annotation (and not just Annotation)
-

NicolasBourassin committed:
Corrections asked by Cha

CharlottePerin committed:
Update views.py
